site/weavedns.md

```diff
@@ -148,8 +148,8 @@ If you want to give the container a name in DNS *other* than its
 hostname, you can register it using the `dns-add` command. For example:
 
 ```bash
-$ C=$(docker run -e WEAVE_CIDR=10.2.1.27/24 -ti ubuntu)
-$ weave dns-add 10.2.1.27 $C -h pingme2.weave.local
+$ C=$(docker run -ti ubuntu)
+$ weave dns-add $C -h pingme2.weave.local
```

Comment:
We should explain all the various `dns-add` permutations. Not necessarily individually, but it must be clear what `dns-add` is capable of.

Also, the `weave dns-remove` example needs updating.